apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -30,8 +72,11 @@
     overflow-y: auto !important;
   }
 </style>
```

Comment:
Oh yeah, I can also throw these into a stylesheet/stylesheets that I can include, rather than just keeping it all in the template. Shouldn't take long!